src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -415,20 +451,17 @@ class FluxPipeline : public DiffusionPipeline {
 
         m_generation_config = ImageGenerationConfig();
 
-        if (m_pipeline_type != PipelineType::IMAGE_2_IMAGE) {
-            m_generation_config.height = transformer_config.m_default_sample_size * vae_scale_factor;
-            m_generation_config.width = transformer_config.m_default_sample_size * vae_scale_factor;
-        }
-
         if (class_name == "FluxPipeline" || class_name == "FluxImg2ImgPipeline" || class_name == "FluxInpaintPipeline" ) {
             if (m_pipeline_type == PipelineType::TEXT_2_IMAGE) {
                 m_generation_config.guidance_scale = 3.5f;
                 m_generation_config.num_inference_steps = 28;
                 m_generation_config.strength = 1.0f;
             } else if (m_pipeline_type == PipelineType::IMAGE_2_IMAGE || m_pipeline_type == PipelineType::INPAINTING) {
-                m_generation_config.guidance_scale = 3.5f;
+                m_generation_config.guidance_scale = 7.0f;
                 m_generation_config.num_inference_steps = 28;
-                m_generation_config.strength = 1.0f;
+                m_generation_config.strength = 0.6f;
+                m_generation_config.height = 1024;
+                m_generation_config.width = 1024;
```

Comment:
should it be computed like here https://github.com/huggingface/diffusers/blob/560fb5f4d65b8593c13e4be50a59b1fd9c2d9992/src/diffusers/pipelines/flux/pipeline_flux_img2img.py#L692-L693? instead of explicit hardcoding 

For some models `vae_scale_factor` can be `1` (see https://huggingface.co/katuni4ka/tiny-random-flux/blob/main/vae/config.json#L5-L7)

Reply:
should you also fix this function ? because it also compute output image size for image2image based on initial image size:
https://github.com/openvinotoolkit/openvino.genai/blob/af41f9c68fe787f25fcbf12dc83e81b69a839a1c/src/cpp/src/image_generation/flux_pipeline.hpp#L392-L407